modules/serverless-functions-quarkus-return-value-types.adoc

```diff
@@ -6,12 +6,10 @@
 [id="serverless-functions-quarkus-return-value-types_{context}"]
 = Permitted types
 
-The input and output types of a function can be any of the following:
+The input and output types of a function can be any of the `void`, `String`, `byte[]`, or primitive types and their wrappers (for example, `int` and `Integer`). Furthermore, they can be certain complex objects.
+
+The possible complex objects include:
```

Comment:
```suggestion
The input and output types of a function can be any of the `void`, `String`, `byte[]`, or primitive types and their wrappers (for example, `int` and `Integer`). They can also be certain complex objects, including the following:
```

Reply:
I've split these paragraphs like I did, because the first paragraph is supposed to be taken and used specially by Jupiter (for SEO or something). So I'll leave it like this.